services/src/main/scala/cromwell/services/metadata/WorkflowQueryParameters.scala

```diff
@@ -60,10 +61,13 @@ object WorkflowQueryParameters {
       rawParameters groupBy { case (key, _) => key.toLowerCase.capitalize }
 
     val Seq(startDate, endDate) = Seq(StartDate, EndDate) map { _.validate(valuesByCanonicalCapitalization) }
+
     val Seq(statuses, names, ids) = Seq(Status, Name, WorkflowQueryKey.Id) map {
       _.validate(valuesByCanonicalCapitalization)
     }
 
+    val Seq(labels) = Seq(WorkflowQueryKey.LabelKeyValue) map { _.validate(valuesByCanonicalCapitalization) }
```

Comment:
canonicalCapitalization here just means take the query parameter keys, turn them all to lowercase and capitalize the first letter, so that these query keys can be matched up to the appropriate WorkflowQueryKey